app/scripts/views/connect_another_device.js

```diff
@@ -0,0 +1,219 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * If the user verifies their email in an instance of Firefox
+ * that other than the one they used to sign up, suggest
+ * that they sign in.
+ */
+define(function (require, exports, module) {
+  'use strict';
+
+  const Cocktail = require('cocktail');
+  const Constants = require('lib/constants');
+  const ExperimentMixin = require('views/mixins/experiment-mixin');
+  const FormView = require('views/form');
+  const Template = require('stache!templates/connect_another_device');
+  const MarketingMixin = require('views/mixins/marketing-mixin');
+  const UserAgent = require('lib/user-agent');
+  const Url = require('lib/url');
+
+  const View = FormView.extend({
+    template: Template,
+
+    events: {
+      'click #signin': '_onSignInClick'
+    },
+
+    beforeRender () {
+      return this.getAccount().isSignedIn()
+        .then((isSignedIn) => {
+          this.model.set('isSignedIn', isSignedIn);
+
+          this.notifier.trigger(`connectAnotherDevice.signedin.${isSignedIn}`);
+          const suffix = this._canSignIn() ? 'eligible' : 'ineligible';
```

Comment:
> I think we try to stay away from ternary in our code base

I didn't realize that, I personally don't mind non-nested ternary operators. OTOH, nested ones are just hard to read. I'm neither for nor against banning ternary, but we decide it's banned, let's add the [no-ternary eslint rule](http://eslint.org/docs/rules/no-ternary) so we can move on to thinking about more important things.